predicators/settings.py

```diff
@@ -112,21 +112,21 @@ class GlobalSettings:
     pybullet_birrt_smooth_amt = 50
     pybullet_birrt_extend_num_interp = 10
     pybullet_control_mode = "position"
-    pybullet_robot_ee_orns = {
-        # Fetch and Panda gripper down and parallel to x-axis.
-        "pybullet_cover": {
+    # env -> robot -> quaternion
+    pybullet_robot_ee_orns = defaultdict(
```

Comment:
I guess specifying rpy is more verbose, if it wasn't the cause of the issue we were talking about